bevy_lint/src/utils/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod hir_parse;
```

Comment:
No I agree. But I recall a long time ago someone suggested we use `pub(crate)` in `bevy_reflect_derive` for all crate-private types so that it's clear at every level that a type isn't exported (and to prevent accidental exports).

I didn't really like it because of the visual noise but I did end up following the advice (as you can tell from looking at the crate lol).

So I don't think there's any right or wrong here. Whatever style you prefer for the project is what I'll stick to :)

Reply:
> No I agree. But I recall a long time ago someone suggested we use `pub(crate)` in `bevy_reflect_derive` for all crate-private types so that it's clear at every level that a type isn't exported (and to prevent accidental exports).

I think I understand the reasoning, but I still prefer removing `pub(crate)` when it is redundant. Thanks for the info!

Reply:
No problem, I'll switch to `pub`